spec/controllers/idv/hybrid_handoff_controller_spec.rb

```diff
@@ -179,6 +183,47 @@
         }.from(false)
       end
     end
+
+    context 'opt in ipp is enabled' do
+      before do
+        stub_up_to(:how_to_verify, idv_session: subject.idv_session)
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        subject.idv_session.service_provider.in_person_proofing_enabled = true
+      end
```

Comment:
What do you think about adding testing when the flags are disabled?

Reply:
I think I can set these to be false initially, and that should cover what we need.